qsiprep/workflows/dwi/merge.py

```diff
@@ -438,7 +443,30 @@ def get_buffernode():
             mem_gb=DEFAULT_MEMORY_MIN_GB,
         )
 
+        dwi_denoise_window = config.workflow.dwi_denoise_window
+        auto_str = ''
+        if denoise_method == 'dwidenoise' and config.workflow.dwi_denoise_window == 'auto':
+            # Configure the denoising window
+            import numpy as np
+
+            dwi_denoise_window = closest_odd(int(np.cbrt(n_volumes)))
+            config.loggers.workflow.info(
+                f'Automatically using {dwi_denoise_window}, {dwi_denoise_window}, '
```

Comment:
What does this boilerplate look like if it's independently denoising a bunch of dwis?

Reply:
I think it'll have separate copies of very similar paragraphs. Users should modify their boilerplates for publication in that case though.